src/App.tsx

```diff
@@ -1,54 +1,74 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
+
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { PersonList } from './components/PersonList';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [query, setQuery] = useState<string>('');
+  const [debounceQuery, setDebounceQuery] = useState<string>('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
  const [debounceQuery, setDebounceQuery] = useState('');
```
it's defined by default since you pass string as initial value